app/controllers/lit/localization_keys_controller.rb

```diff
@@ -104,28 +101,30 @@ def valid_keys
     end
 
     def grouped_localizations
-      @_grouped_localizations ||= begin
-        {}.tap do |hash|
-          @localization_keys.each do |lk|
-            hash[lk] = {}
-            lk.localizations.each do |l|
-              hash[lk][l.locale.locale.to_sym] = l
+      @_grouped_localizations ||=
+        begin
+          {}.tap do |hash|
+            @localization_keys.each do |lk|
+              hash[lk] = {}
+              lk.localizations.each { |l| hash[lk][l.locale.locale.to_sym] = l }
             end
           end
         end
-      end
     end
 
     def localization_for(locale, localization_key)
       @_localization_for ||= {}
       key = [locale, localization_key]
       ret = @_localization_for[key]
+      binding.pry if localization_key == 'scopes.string'
       if ret == false
         nil
       elsif ret.nil?
         ret = grouped_localizations[localization_key][locale]
         unless ret
+          ::Rails.logger.info "******** REFRESHING the key"
```

Comment:
Is it part of previous PR when you tried to reproduce `ret` being false or is it relevant for this PR?